src/test/java/plugins/LdapPluginTest.java

```diff
@@ -73,6 +90,24 @@ public void login_ok() {
     }
 
     /**
+     * Scenario: Login with ldap uid and password
+     * Given I have a docker fixture "ldap" that allows anonymous binding
+     * And Jenkins is using ldap as security realm
+     * When I login with user "jenkins" and password "root" without manager credentials
+     * Then I will be successfully logged on as user "jenkins"
```

Comment:
Sorry, my comment was related to the Gherkin-like comments and not the test itself. It just duplicates whatever the test is doing anyway. If the test is hard to understand(which it is not in this case), we should fix the framework/POs so it is cloear what is going on and not to duplicate the information in text.